Tasks/KubernetesManifestV0/src/actions/bake.ts

```diff
@@ -109,7 +109,7 @@ export async function bake(ignoreSslErrors?: boolean) {
     const renderType = tl.getInput('renderType', true);
     let renderEngine: RenderEngine;
     switch (renderType) {
-        case 'helm2':
+        case 'helm':
```

Comment:
You'll need to. add `case :  helm2' as well for back compatibility

Reply:
Good catch!